xrpl4j-core/src/main/java/org/xrpl/xrpl4j/codec/binary/types/AmountType.java

```diff
@@ -251,8 +300,14 @@ public JsonNode toJson() {
    * @return {@code true} if this AmountType is native; {@code false} otherwise.
    */
   private boolean isNative() {
-    // 1st bit in 1st byte is set to 0 for native XRP
-    return (toBytes()[0] & 0x80) == 0;
+    // 1st bit in 1st byte is set to 0 for native XRP, 3rd bit is also 0.
+    // 0xA0 is 1010 0000
+    return (toBytes()[0] & 0xA0) == 0;
+  }
+
+  private boolean isMpt() {
+    // 1st bit in 1st byte is 0, 2nd bit is 1, and 3rd bit is 1
```

Comment:
The binary encoding supports encoding a negative MPT amount, so we should probably omit references to the second bit:

```suggestion
    // To be an MPT, 1st bit in 1st byte is always 0; 3rd bit is always 1
```

Reply:
changed in https://github.com/XRPLF/xrpl4j/pull/556/commits/73e282ed78fa791fd33d801ca76174b65eae42bf